pacman/executor/injection_decorator.py

```diff
@@ -18,6 +18,7 @@ def supports_injection(injectable_class):
     orig_init = injectable_class.__init__
 
     def new_init(self, *args, **kwargs):
+        # pylint: disable=protected-access
         orig_init(self, *args, **kwargs)
         for method in injectable_class.__dict__.itervalues():
```

Comment:
wow, this defintately should be a porperty, not a call to the private __dict__!

Reply:
This is acceptable in this context - decorators sometimes have to do some dirty work!